packages/ors/index.ts

```diff
@@ -25,6 +25,14 @@ import {
 } from "./parameters"
 import { decode } from "@googlemaps/polyline-codec"
 
+//ORS API returns an error code along with a message in the response body when an error occurs.
+interface ORSErrorProps{
```

Comment:
Same as with GraphHopper: remember to type the error response we get from ORS, and have `ORSErrorProps` extend from `ErrorProps`